pkgs/development/python-modules/catppuccin-catwalk/default.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pillow,
+}:
+buildPythonPackage rec {
+  pname = "catppuccin-catwalk";
+  version = "0.4.0";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "catppuccin_catwalk";
+    sha256 = "sha256-5TAw5H3soxe9vLhfj1qs8uMr4ybrHlCj4zdsMzvPo6s=";
```

```suggestion
    hash = "sha256-5TAw5H3soxe9vLhfj1qs8uMr4ybrHlCj4zdsMzvPo6s=";
```
We should use SRI hash.
https://nixos.org/manual/nixpkgs/stable/#fetchurl